src/biokbase/narrative/common/util.py

```diff
@@ -91,24 +94,3 @@ def _user(self):
 # Get/set KBase environment variables by getting/setting
 # attributes of this object.
 kbase_env = _KBaseEnv()
-
-
-class BuildDocumentation(Command):
```

Comment:
old BuildDocumentation class. It is only referenced in the setup file (see below)